openapi.yaml

```diff
@@ -208,10 +435,345 @@ definitions:
+      type:
+        $ref: '#/definitions/PlayerType'
+      attributes:
+        type: object
+        properties:
+          memberName:
+            $ref: '#/definitions/MemberName'
+          memberLevel:
+            $ref: '#/definitions/MemberLevel'
+          memberExpAboveLevel:
+            $ref: '#/definitions/MemberExpAboveLevel'
+          playerBet:
+            $ref: '#/definitions/PlayerBet'
+          playerStatus:
+            $ref: '#/definitions/PlayerStatus'
+          playerCards:
+            $ref: '#/definitions/Cards'
+  PlayerInGameResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/PlayerInGameResource'
+  PlayerInGameResults:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/PlayerInGameResource'
+  MemberLevel:
+    type: integer
+    minimum: 1
+    description: Player's level in the system.
+    example: 20
+  MemberName:
+    type: string
+    example: Jack
+  MemberPassword:
+    type: string
+    example:  hunter2
+    format: password
+  MemberPatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/PlayerId'
+          type:
+            $ref: '#/definitions/PlayerType'
+          attributes:
+            type: object
+            properties:
+              memberName:
+                $ref: '#/definitions/MemberName'
+              memberEmail:
+                $ref: '#/definitions/MemberEmail'
+              memberPassword:
+                $ref: '#/definitions/MemberPassword'
+              memberLevel:
+                $ref: '#/definitions/MemberLevel'
+              memberExpAboveLevel:
+                $ref: '#/definitions/MemberExpAboveLevel'
+              gameIds:
+                $ref: '#/definitions/GameIds'
+            additionalProperties: false
+        required: 
+          - id
+          - type
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  MemberPostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/PlayerType'
+          attributes:
+            type: object
+            properties:
+              memberName:
+                $ref: '#/definitions/MemberName'
+              memberEmail:
+                $ref: '#/definitions/MemberEmail'
+              memberPassword:
+                $ref: '#/definitions/MemberPassword'
+            required:
+              - memberName
+              - memberEmail
+              - memberPassword
+            additionalProperties: false
+        required: 
+          - type
+          - attributes
+        additionalProperties: false
+    required: 
+      - data
+    additionalProperties: false 
+  MemberResource:
+    properties:
+      id:
+        $ref: '#/definitions/PlayerId'
```

Comment:
Since a `member` can be a part of multiple `games`, the `member` resource will have to have a different id than the `player` resource. @tsoliangwu0130 suggested the player resource should have an id like `{gameId}-{memberId}` or something similar.